modules/dynamic-plug-in-development.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-get-started.adoc
+
+:_content-type: PROCEDURE
+[id="dynamic-plugin-development_{context}"]
+= Dynamic plug-in development
+
+You can run the plug-in using a local development environment. The {product-title} web console runs in a container connected to the cluster you have logged into.
+
+.Prerequisites
+* You must have an OpenShift cluster running.
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have link:https://yarnpkg.com/[`yarn`] installed.
+* You must have link:https://www.docker.com/[Docker] v3.2.0 or newer or link:https://podman.io/[Podman] installed and running.
+
+.Procedure
+
+. In your terminal, run the following command to install yarn.
```

Comment:
Do I need to install yarn? It is listed as a prereq. Again, I am not target audience; maybe my ignorance.

Reply:
This isn't actually installing yarn. This is installing dependencies for your plugin using yarn, which manages dependencies for frontend apps. yarn itself you must have previously installed as one of the prereqs.

Reply:
I have added this clarification to the `yarn install `command.